app/core/domain/email_manager_test.py

```diff
@@ -24,14 +24,33 @@ class SendEmailToAdminTest(app_engine_test_base.GenericTestBase):
     """Test for email to be sent to admin email address."""
 
     def test_send_mail_to_admin(self):
-        """Test send_mail_to_admin sends mail to admin email address."""
+        """Test send_mail sends mail to admin email address."""
```

Comment:
This test only works with the send_mail_to_admin method right?

Reply:
Ah yes. I've fixed the comment, thanks!